docs/USING_BULK_SELECTION.md

```diff
@@ -0,0 +1,161 @@
+# Bulk & Selection Plugins
+
+The Bulk and Selection plugins compliment each other. After selecting multiple rows, the user will most likely want to perform a bulk action on the selected rows such as altering them to have the same value. 
+
+The selection plugin turns the first column into checkboxes so that rows can be selected and unselected. The selected rows are captured in the selection state. 
+
+#### grid vs dataSource vs selection reducers
+
+const columnData = this.props.grid.get('sticky');
+console.log(JSON.stringify(columnData) ); <-- only column data
```

Comment:
Since you can console.log complex objects, we probably don't need the `JSON.stringify` here.